gc/base/CycleState.hpp

```diff
@@ -51,6 +51,7 @@ class MM_CycleState {
 
 	volatile bool _finalizationRequired; /**< Does the current cycle require finalization upon completion */
 	volatile bool _dynamicClassUnloadingEnabled; /**< Determine if dynamic class unloading is enabled for this cycle */
+	volatile bool _noCompactionAfterSweep; /* There is no compaction after sweep for this cycle */
```

Comment:
This is a VLHGC specific flag and ideally should go to existing MM_CycleStateVLHGC class. But since this class already violates that rule with a few other VLHGC fields (and it would be non-trivial to move them), this is OK.